mlrun/frameworks/lightgbm/__init__.py

```diff
@@ -0,0 +1,53 @@
+"""
+Description:
+__init__ function of LightGBM-autologger. Will be extended and contain multiple LightGBM-specific functions.
+"""
+
+import mlrun
+from mlrun.frameworks._common.pkl_model_server import PickleModelServer
+from mlrun.frameworks.mlbase.mlrun_interface import MLBaseMLRunInterface
+
+# Temporary placeholder, LGBMModelServer may deviate from PklModelServer in upcoming versions.
+LGBMModelServer = PickleModelServer
+
+
+def apply_mlrun(
+    model,
+    context: mlrun.MLClientCtx = None,
+    X_test=None,
+    y_test=None,
+    model_name=None,
+    generate_test_set=True,
+    feature_vector = None,
+    **kwargs
+):
+    """
```

Comment:
Make sure the parameters are aligned to the order of the parameters in the function's header. Also, please align the spaces.